app/models/problem_report.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: problem_reports
+#
+#  id                  :integer          not null, primary key
+#  reporter_user_id    :integer
+#  last_update_user_id :integer
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  venue               :string
+#  computer_number     :string
+#  description         :string
+#  is_critical         :boolean
+#  is_fixed            :boolean
+#  is_fixable          :boolean
+#  remarks             :string
+#
+# Indexes
+#
+#  index_problem_reports_on_last_update_user_id  (last_update_user_id)
+#  index_problem_reports_on_reporter_user_id     (reporter_user_id)
+#
+
+class ProblemReport < ApplicationRecord
+  belongs_to :reporter_user, :class_name => 'User'
```

Comment:
I think you can just do `User` to refer to the class and not the string

Reply:
Eh but every example on the internet uses string

Reply:
I used string too in Timeslot last time